include/odp/api/spec/classification.h

```diff
@@ -176,6 +176,27 @@ typedef struct odp_bp_param_t {
 	odp_threshold_t threshold;
 } odp_bp_param_t;
 
+/**
+ * Classifier CoS specific statistics counters
+ *
+ * Counters are incremented per packet classified to the CoS.
+ */
+typedef struct odp_cls_cos_stats_t {
+	/** Number of octets in classified packets. In case of Ethernet, packet
+	 * size includes MAC header. */
+	uint64_t octets;
+
+	/** Number of classified packets. */
+	uint64_t packets;
+
+	/** Number of packets discarded due to a drop action. */
+	uint64_t discards;
```

Comment:
Another option would be to increment packets/octets normally also in case of drop CoS. Application knows which CoSes are setup with drop and enqueue actions.

Reply:
In v9.